src/adaptors/http2/http2_adaptor.c

```diff
@@ -449,6 +469,17 @@ static qdr_http2_stream_data_t *create_http2_stream_data(qdr_http2_session_data_
 }
 
 
+/**
+ * This callback function  is invoked when the nghttp2 library tells the application about the error code, and error message.
+ */
+static int on_error_callback(nghttp2_session *session, int lib_error_code, const char *msg, size_t len, void *user_data)
+{
+	qdr_http2_connection_t *conn = (qdr_http2_connection_t *)user_data;
+	qd_log(http2_adaptor->protocol_log_source, QD_LOG_TRACE, "[C%"PRIu64"] Error generated in the on_error_callback, lib_error_code=%i, error_msg=%s", conn->conn_id, lib_error_code, msg);
```

Comment:
If this results in a user-visible failure, should the log level be warn/error/etc?

Reply:
Changed log level to ERROR